### ISSUE TITLE
Default & Straight buttons to Configure "Laser Smoothing numbers"

### ISSUE THREAD

**Issue by Reiko9** [Frooxius/NeosPublic#554]
As of the Moment if a User Forgets the "Default" configuration numbers for Laser smoothing after modifying settings they can be stuck with a laser that sticks to items worse than bubblegum..

A "Default" button simply enters the default numbers into the textboxes used to configure the laser.

A "Straight" button enters the values to obtain a straight line...
   These Experimental values from Zane have worked well enough for me:
Smooth Speed: 50
Modulate Start Angle: 0.2
Modulate End Angle: 0.2
Modulate Exponent: 1.0
Modulate Speed Multiplier: 50
Stick Threshold: 0.00

This is the simplest method I can find to keep many older users happy and reduce the head ache for new users who dare play with the settings.  (Literally just pastes numbers into the textboxes :)

**Comment by TehTurk** [comment]
While these would be good, I don't know if it'd be reasonable in the current timeframe. Up next is the dash so I think alot of the laser control would be moved around as well too.

**Comment by shiftyscales** [comment]
Yeah, these won't going to be added onto the current UI. I'm not sure that's the best approach either as it's very specific. I think a more universal/powerful solution could be something like "settings presets" not just for lasers, but any arbitrary settings you wish to link together.
